src/test/java/bio/terra/service/snapshot/SnapshotServiceTest.java

```diff
@@ -490,7 +499,6 @@ public void listAuthorizedSnapshotsWhenEcmThrows() throws Exception {
         "ECM exception should not block snapshot enumeration",
         service.listAuthorizedSnapshots(TEST_USER, errors),
         equalTo(samAuthorizedSnapshots));
-    verify(ecmService, times(1)).getRasDbgapPermissions(TEST_USER);
```

Comment:
Why is this removed?

Reply:
Now that this test is using junit 5 with strict stubbing, this `verify()` is redundant since it's verifying a mock from L484 above. If the mock wasn't used, the test would fail, so `verify()` is no longer necessary.